addons/website_sale_loyalty/views/loyalty_program_views.xml

```diff
@@ -21,6 +21,16 @@
             <xpath expr="//div[@id='o_loyalty_program_availabilities']" position="after">
                 <field name="website_id" invisible="not ecommerce_ok" options="{'no_create': True}" groups="website.group_multi_website" placeholder="All websites"/>
             </xpath>
+            <div id="trigger_products" position="inside">
+                <div
+                    invisible="program_type!='ewallet'"
+                    class="alert alert-warning mt-1"
+                    role="alert"
+                >
+                    To be able to top up eWallet from the portal, eWallet products must be
+                     published
+                </div>
+            </div>
```

Comment:
This warning would be better if dynamic.
Like: no product set or products set with at least one that's non published ==> show warning.
But if all the products set are published, then why showing the warning?